### PR TITLE
Score place change

### PR DESCRIPTION
**Pull request by PillouMatou**
Places the score at the top left of the game

### CODE REVIEW

File: gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java [hunk 1]
```diff
@@ -371,7 +375,11 @@ private void enter(final Target t) {
         text.setText(cst);
 
         final String soundResource = "data/race/sounds/frog.WAV";
-        gameContext.getSoundManager().add(soundResource);
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java]: Why did you change back the sound to ForegroundSoundsUtils.playSound ? you should keep using the soundManager to play sounds.

File: gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java [hunk 2]
```diff
@@ -133,6 +134,7 @@ public Race(final IGameContext gameContext, final Stats stats, final String type
                         ((Target) e.getTarget()).done = true;
                         enter((Target) e.getTarget());
                         stats.incrementNumberOfGoalsReached();
+                        stats.notifyNewRoundReady();
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java]: Why did you change the newRoundReady position ? It should be called at the launch of a game in order to start the stats and take a screenshot of the game for stats context. We should not be able to exit the game while the notifyNewRoundReady is called, what could happened here if you're not entering an object before exiting.

File: gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java [hunk 3]
```diff
@@ -372,6 +376,11 @@ private void enter(final Target t) {
 
         final String soundResource = "data/race/sounds/frog.WAV";
         gameContext.getSoundManager().add(soundResource);
+        try {
+            ForegroundSoundsUtils.playSound(soundResource);
+        } catch (final Exception e) {
+            log.warn("Can't play sound: no associated sound : " + e.toString());
+        }
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java]: You also need to remove this part (it's done by the soundManager now)

File: gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java [hunk 4]
```diff
@@ -21,6 +21,7 @@
 import net.gazeplay.IGameContext;
 import net.gazeplay.commons.configuration.BackgroundStyleVisitor;
 import net.gazeplay.commons.gaze.devicemanager.GazeEvent;
+import net.gazeplay.commons.utils.games.ForegroundSoundsUtils;
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java]: this import is not necessary

**Comment by schwabdidier** [comment]
Please @PillouMatou modify that when you come back to internship

**Comment by PillouMatou** [comment]
> Please @PillouMatou modify that when you come back to internship

I made the changes requested 13 days ago, it's the second commit.

**Comment by schwabdidier** [comment]
I am not sure, @kuhlkrein's comments seem to be latest. But I see this comment `Add more commits by pushing to the UpdateFrogGame branch on PillouMatou/GazePlay-1.`, I don't see exactly what does it mean.

Can you check it with @kuhlkrein next time ?

**Comment by PillouMatou** [comment]
> I am not sure, @kuhlkrein's comments seem to be latest. But I see this comment `Add more commits by pushing to the UpdateFrogGame branch on PillouMatou/GazePlay-1.`, I don't see exactly what does it mean.
> 
> Can you check it with @kuhlkrein next time ?

Ok we will check it tomorrow.
